scripts/build_android.sh

```diff
@@ -86,6 +86,21 @@ CMAKE_ARGS+=("-DUSE_MOBILE_OPENGL=OFF")
 # Use-specified CMake arguments go last to allow overridding defaults
 CMAKE_ARGS+=($@)
 
+USE_GCC=true
+for arg in ${CMAKE_ARGS[@]};do
+  if [ $arg == "-DANDROID_TOOLCHAIN=clang" ];then
+    USE_GCC=false
+  elif [ $arg == "-DANDROID_TOOLCHAIN=gcc" ];then
+    USE_GCC=true
+  fi
+done
+
+if $USE_GCC;then
+  CMAKE_ARGS+=("-DCMAKE_CXX_FLAGS=-s")
+else
+  CMAKE_ARGS+=("-DCMAKE_EXE_LINKER_FLAGS=-s")
+fi
+
```

Comment:
Building with `-s` seems reasonable to me, but it should be handled in `CMakeLists.txt` rather than in the build script. We don't expect that all users will use `scripts/build_android.sh`, this script is more of an example.

Reply:
OK, I'll more them to some cmake files.